.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: precise
```

Comment:
rather than the commit that edits this .travis.yml I suggest pulling in the latest from master to see if this fixes your issues. I do see that currently its simply failing with 7 findbugs errors.